.github/workflows/test.yaml

```diff
@@ -0,0 +1,43 @@
+name: Test Charts
+
+on: pull_request
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.x
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2
+
+      - name: Run chart-testing (list-changed)
+        id: list-changed
+        run: |
+          changed=$(ct list-changed --target-branch ${{ github.event.repository.default_branch }})
```

Comment:
Should this be `github.base_ref` (which is the "target branch of the pull request in a workflow run") instead of `repository.default_branch`?

Reply:
I'm not sure. It sounds very reasonable to use `github.base_ref`, but by default `ct` uses default branch to identify changed charts:
- https://github.com/helm/chart-testing/blob/a562b39ed8d84ec2fd2478d85549240b47847f28/doc/ct_list-changed.md#options
- https://github.com/helm/chart-testing/blob/a562b39ed8d84ec2fd2478d85549240b47847f28/doc/ct_install.md#options

I think it's typical of helm chart repositories to have PRs into default branch only. So, `--target-branch` flag can be deleted ("main" is a default value for the flag) to keep current behavior with less code or changed to `github.base_ref`. 

What do you think?

Reply:
(Based on the docs) these two work the same way when targeting `main`, but using `github.base_ref` is more flexible.  For example we might need to create another branch for maintenance release, which we still want to target with PRs.

Reply:
Changed target branch to `github.base_ref`.